mailbox/lucene/src/test/java/org/apache/james/mailbox/lucene/search/LuceneMailboxMessageSearchIndexTest.java

```diff
@@ -606,14 +606,13 @@ public void notOperatorShouldReverseMatching() throws Exception {
     }
     
     private final class SimpleMailbox implements Mailbox {
-        private TestId id;
+        private final TestId id;
 
         public SimpleMailbox(long id) {
         	this.id = TestId.of(id);
         }
 
         public void setMailboxId(MailboxId id) {
-            this.id = (TestId)id;
```

Comment:
Sorry, I'm note sure for this part of the fixup. Wha's the point of an empty setter?